blackrock/src/config/config.json

```diff
@@ -0,0 +1,21 @@
+
+
+{
+  "development": {
+    "username": "blackrockadmin",
+    "password": "admin1234",
+    "database": "taskmasterdb",
+    "host": "taskmasterdb.cepyxa3at9gn.us-east-1.rds.amazonaws.com",
```

Comment:
We are using postgres for backend, not AWS

Reply:
@joetta25 OK I see.  Could you pull develop into this branch and/or manually correct this config.json with the proper credentials for the local postgres db or commit an empty version of this file so that individuals can put in the configuration to their local databases?